content/en/docs/ops/deployment/performance-and-scalability/index.md

```diff
@@ -109,21 +109,21 @@ average and tail latencies. The actual tail latency depends on the traffic patte
 ### Latency for Istio {{< istio_release_name >}}
 
 Inside the mesh, a request traverses the client-side proxy and then the server-side
-proxy. In the default configuration of Istio {{< istio_release_name >}} (i.e. Istio with telemetry v2), the two proxies add about 2.8 ms and 2.7 ms to the 90th and 99th percentile latency, respectively, over the baseline data plane latency.
```

Comment:
carolyn, can you point to / provide flame graphs between 1.5 and 1.6, so that we can account this reduction.
@bianpengyuan @lambdai can you follow up on this?
We will not merge this PR until we know that.

Reply:
sure!

Reply:
Is this with or without telemetry feature?

Reply:
I don't see a 3.1 ms gap in the graph above..

Reply:
@bianpengyuan 
(1) yes, with telemetry enabled, as stated `in the default configuration (which is with telemetryv2 enabled)`.  
(2) check the difference of (pinkline minus greenline) for conn=16.

Reply:
Do we have the same graph for 1.5? want to know where the latency increasing is from?

Reply:
yes, istio.io 1.5 release is those graphs: https://istio.io/docs/ops/deployment/performance-and-scalability/